README.md

```diff
@@ -7,22 +7,35 @@
 <!--- end of master only -->
 
 # Calico DCOS
-This repo contains the implementation of the Calico Universe package for
+This repo contains the source code implementation of the Calico Universe package for
 DCOS.
 
-If you would like to install Calico on your DCOS cluster, please refer 
-to the [DCOS Calico Universe installation](https://github.com/projectcalico/calico-containers/blob/master/docs/mesos/DCOS.md) guide.  This guide describes
-how to install Calico on your DCOS deployment and provides details for
-using Calico in that deployment.
+If you would simply like to install Calico on your DCOS cluster, please refer 
+to the [DCOS Calico Universe installation](https://github.com/projectcalico/calico-containers/blob/master/docs/mesos/DCOS.md) guide, which describes
+how to install Calico on your DCOS cluster and provides details for
+using Calico.
+
+Below are instructions to build the Calico Universe plugin from source.
+
+### Build
```

Comment:
I think we should build steps out of the readme for several reasons:
1. For a reader, knowing the build process isn't very useful if they don't also know the deployment process for testing it. And it is not in our interest to spend time documenting that full process right now.
2. The makefile is supposed to be self documenting, or at least show the general idea of how this is done. 
3. The steps as you've documented need improvement. We should do something like Universe does where you pass in `$DOCKER_TAG` instead of retagging the image afterwards as you've shown here. Again, not in our interest to spend time improving this right now.